### PR TITLE
config(sps23): Configure Swiss Python Summit 2023 event

### PR DESCRIPTION
**Pull request by raoulb**

### CODE REVIEW

**Comment by raoulb** [comment]
Any review or news here?

File: configs/conferences/swisspythonsummit2023/config.php [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+	/**
+	 * Abschnitte aud der Startseite und darunter aufgeführte Räume
+	 * Es können beliebig neue Gruppen und Räume hinzugefügt werden
+	 *
+	 * Die Räume müssen in $CONFIG['ROOMS'] konfiguriert werden,
+	 * sonst werden sie nicht angezeigt.
+	 */
+	'GROUPS' => array(
+		'Lecture Rooms' => array(
+			'Aula',
```
Comment (jbeyerstedt) [inline configs/conferences/swisspythonsummit2023/config.php:116]: This should be the same as in the `$ONFIG['ROOMS']` array. So either use "weissersaal" here or "Aula" in line 125.

Comment (raoulb) [reply configs/conferences/swisspythonsummit2023/config.php:116]: Oh, good catch, thanks. Copy & paste error from DGWK23.

**Comment by jbeyerstedt** [comment]
> Any review or news here?

Please poke whoever has added the streaming endpoints or ask in the IRC channel to merge and deploy this. I don't see this event in our wiki, so I don't know who is your contact at VOC for the event.

**Pull request merged**